headers.go

```diff
@@ -471,6 +497,25 @@ func validateHeaderParameters(h map[any]any, protected bool) error {
 			if err := ensureCritical(value, h); err != nil {
 				return fmt.Errorf("header parameter: crit: %w", err)
 			}
+		case HeaderLabelType:
+			is_tstr := canTstr(value)
```

Comment:
`is_tstr` => `isTstr`

Reply:
I've changed this and the isAlg for consistency